RInAppMessaging/Classes/CampaignDispatcher.swift

```diff
@@ -53,57 +61,59 @@ internal class CampaignDispatcher: CampaignDispatcherType {
         }
     }
 
+    /// must be executed on `dispatchQueue`
```

Comment:
Can this be asserted for in debug builds programmatically?

Reply:
It seems that currently there is no way to check if something is executed on specific queue. `dispatch_get_current_queue()` is not available anymore and there is no replacement for it.